encipher/lib/encipherio.js

```diff
@@ -926,6 +959,32 @@ let legoEptCloud = class {
             await era.addStateEvent("box_state","websocket",0);
             this.offlineEventFired = false;
           }
+          this.disconnectCloud = false;
+          const keys = await rclient.scanResults("notification:resend:*", 1000)
```

Comment:
Try using zset to store notifications and use timestamp as score. You can get notification history by limiting number of time range.